UI.NUnitVersion/TestData/HearingTestData.cs

```diff
@@ -47,4 +47,43 @@ public static BookingDto CreateHearingDto(bool remote = false, DateTime? schedul
         };
         return bookingDto;
     }
+
+    public static BookingDto CreateHearingEndPoint(bool remote = false, DateTime? scheduledDateTime = null)
```

Comment:
Please add a description to the method so it is clear why the returning object differs to the method above.
Also the name might be more meaningful as `CreateHearingWithAnEndpoint`.

Reply:
Thus the Participant date for end point